src/exodus/createpayload.cpp

```diff
@@ -549,5 +549,25 @@ std::vector<unsigned char> CreatePayload_ExodusAlert(uint16_t alertType, uint32_
     return payload;
 }
 
+std::vector<unsigned char> CreatePayload_CreateDenomination(uint32_t propertyId, uint64_t value)
+{
+    std::vector<unsigned char> payload;
+
+    uint16_t messageType = 1025;
```

Comment:
better to use `EXODUS_TYPE_CREATE_DENOMINATION` instead.

Reply:
The problems is it will look ugly because other functions in the files hard-coded like this. But I agree we should not hard-coded like this so I will fix all of other functions.